exercises/practice/alphametics/AlphameticsTests.fs

```diff
@@ -78,10 +84,11 @@ let ``Puzzle with seven letters`` () =
           ('T', 7) ]
         |> Map.ofList
         |> Some
-    solve puzzle |> should equal expected
+    return solve puzzle |> should equal expected
+    }
 
-[<Fact(Skip = "Remove this Skip property to run this test")>]
-let ``Puzzle with eight letters`` () =
+[<Fact(Timeout=2000,Skip = "Remove this Skip property to run this test")>]
```

Comment:
Nice idea. The actual timeouts are slightly tricky to determine, but I'm fine with having some value here. That said, we currently can't merge this as the test runner will strip the attribute before it runs the tests:

```
type EnableAllTests() =
    inherit SyntaxVisitor()

    override _.VisitSynAttribute(attr: SynAttribute) : SynAttribute =
        match attr.TypeName with
        | LongIdentWithDots ([ ident ], _) when ident.idText = "Fact" ->
            base.VisitSynAttribute
                { attr with
                      ArgExpr = SynExpr.Const(SynConst.Unit, attr.ArgExpr.Range) }
        | _ -> base.VisitSynAttribute(attr)
```

Would you be interested in fixing that too?

Reply:
Yup will have a look at that shortly.

Reply:
Will look at this. Which repo and file is it?

Reply:
I think timeouts should be discussed at the practise exercise specification, hence my other PR to update canonical json with a timeout property. This might include relative timings between tests as implied here between 8 versus 10 letter tests. The actual implementation would depend on the track maintainers factoring in the built in online test runner timeouts and language performance etc. I would probably default to 10 seconds per test, _a propos_ of anything more specific.

Reply:
I already wrote a reply but it has inexplicably disappeared. What files do I need to look at?

Reply:
See https://github.com/exercism/fsharp/pull/1029#issuecomment-964015126. I should have posted that comment here.

Reply:
@martinfreedman Let me know if you need any help/pointers. The F# syntax tree can be tricky to work with.

Reply:
Will do. And yes the Diamond exercise does not use `[<Fact>]`! I will add this into my solution when I have a chance (I have Covid right now). Any other exercises tests with non-standard Attributes?

Reply:
> (I have Covid right now)

Oh no! All the best.

> Any other exercises tests with non-standard Attributes?

The `simple-linked-list` exercise uses a `[<Theory>]` attribute (which should actually be converted to multiple `[<Fact>]` attributes.